test/test_gc.c

```diff
@@ -317,8 +317,8 @@ static char* test_gc_static_allocation()
 {
     DTOR_COUNT = 0;
     GarbageCollector gc_;
-    void *bos = __builtin_frame_address(0);
-    gc_start(&gc_, bos);
+    void *bos = NULL;
+    gc_start(&gc_, &bos);
```

Comment:
This should not make a difference to the test. Does it?

Reply:
> The frame address is normally the address of the first word pushed on to the stack by the function. However, the exact definition depends upon the processor and the calling convention

oddly enough, gcc in most 64 bit Intels, it does not seem to be the case, so bos goes bellow whatever that function is returning, thus maintaining, bos, var, tos order.